test/terra/backends/qasm_simulator/qasm_noise.py

```diff
@@ -148,3 +150,26 @@ def test_kraus_gate_noise(self):
                 **self.BACKEND_OPTS).result()
             self.assertSuccess(result)
             self.compare_counts(result, [circuit], [target], delta=0.05 * shots)
+
+class QasmKrausNoiseOnQFTTests:
+    """QasmSimulator Kraus error noise model on QFT tests."""
+
+    SIMULATOR = QasmSimulator()
+    BACKEND_OPTS = {}
+
+    def test_kraus_gate_noise_on_QFT(self):
+        """Test Kraus noise on a QFT circuit"""
+
+        shots = 1000
+        noise_models = ref_kraus_noise.kraus_gate_error_noise_models_full()
+
+        for noise_model in noise_models:
```

Comment:
Only one model is returned by `kraus_gate_error_noise_models_full`

Reply:
You are correct, I just did it this way to be consistent with the other tests. Do you think I should turn it into a single noise_model?

Reply:
I think so